src/components/WebexAvatar/WebexAvatar.stories.js

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
+import {map} from 'rxjs/operators';
 import {storiesOf} from '@storybook/react';
 
+import PeopleJSONAdapter from '../../adapters/PeopleJSONAdapter';
+import person from '../../data/person';
+
 import WebexAvatar from './WebexAvatar';
 
-storiesOf('Webex Avatar', module).add('Avatar with title', () => <WebexAvatar title="Tom Smith" />);
+const stories = storiesOf('Webex Avatar', module);
+const peopleJSONAdapter = new PeopleJSONAdapter();
+
+stories.add('With appropriate PersonID', () => <WebexAvatar personID={person.id} adapter={peopleJSONAdapter} />);
+
+stories.add('With inappropriate PersonID', () => <WebexAvatar personID="Wrong personID" adapter={peopleJSONAdapter} />);
+
+stories.add('With different data', () => {
+  peopleJSONAdapter.getPerson(person.id).pipe(map((data) => ({...data, status: 'ooo'}))); // Mapped Observable
```

Comment:
we need to somehow pass this mapped observable to the component.

Reply:
With @adamweeks's idea of passing in the JSON object to the JSON adapter class, I think that solves the issue of how to "change data".
We would have a story for each of the UI states, and we would just modify the JSON object `status` before passing it to `PeopleJSONAdapter`.